velox/type/Type.h

```diff
@@ -1533,6 +1556,11 @@ bool typeExists(const std::string& name);
 /// child types.
 TypePtr getType(const std::string& name, std::vector<TypePtr> childTypes);
 
+/// Return the custom cast operator for the custom type with the specified name.
+/// This function returns nullptr if a type with the specified name does not
```

Comment:
nit: drop "This function" for brevity: "Returns nullptr ..."

Reply:
Revised the comment. Thank you for the suggesiton!